doc/dev/Mgmt-Polishing-Configurations.md

```diff
@@ -0,0 +1,355 @@
+# Azure SDK for .NET - Management Plane SDK Polishing Configurations
+
+To ensure the quality of the Azure SDK for .NET and adherence to the [naming conventions](https://github.com/Azure/azure-sdk-for-net/tree/main/doc/dev/Mgmt-Naming-Conventions.md), the CSharp CodeGen offers flexible configurations. These configurations allow for changes to the names of classes, methods, properties, and parameters, as well as the data types of properties and parameters.
+
+# Guidelines for Data Type Changes
+
+In the API specifications, many properties and parameters are defined as string types but have special formats in .NET. We need to correct these types in the Azure .NET SDK.
+
+- The data type of a property should be `uuid` if its name is end of `Id` / `Guid` and value is really a uuid.
+
+- The data type of a property should be `ResourceIdentifier` if its name is end of `Id` and value is really a ResourceIdentifier.
+
+- The data type of a property should be `ResourceType` if its name is `ResourceType` or end with `Type` and the value is really a ResourceType('Microsoft.xxxxx/xxxx').
+- The data type of a property should be `ETag` if its name is `etag` and the value is really a ETag. The property name  should be `ETag`(Optional).
+
+- The data type of a property might be `AzureLocation` if its name is end with `location` / `locations` .
+
+- `Binary` / `String` property might change to `BinaryData` type.
+
+- Pay attention to models similar to common type models and confirm whether they really can't be replaced by the common types from ResourceManager(Optional). 
+
+# Guidelines for Operation Changes
+
+- Put / patch operation, optional body parameter should be changed to required.
+
+- All `ListOperations` method should be removed because it's not useful in SDK and the SDK itself can show all the available operations in it through public APIs. 
+
```

Comment:
should we include the `GetOperationStatus` operation should not be included in the generated library as well?